src/plugins/InspectorPlugin.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Inspector from "@open-rpc/inspector";
+import { Grid } from "@material-ui/core";
+import { IMethodPluginProps } from "@open-rpc/docs-react/build/Methods/Methods";
+import UISchemaStore from "../stores/UISchemaStore";
+import { IUISchema } from "../UISchema";
+import searchBarStore from "../stores/searchBarStore";
+import { ExamplePairingObject, ExampleObject } from "@open-rpc/meta-schema";
+
+const InspectorPlugin: React.FC<IMethodPluginProps> = (props) => {
+  const [UISchema]: [IUISchema] = UISchemaStore();
+  const [searchUrl, { results, error }, setSearchUrl] = searchBarStore();
+  const method = props.openrpcMethodObject;
+  const examplePosition = 0;
+  let example;
+  let exampleParams;
+  if (method && method.examples && method.examples[examplePosition]) {
+    example = method.examples[examplePosition] as ExamplePairingObject;
+    exampleParams = (example.params as ExampleObject[]).map((p) => p.value);
+  }
+  return (
+    <Grid style={{ height: "300px", width: "100%", overflowY: "auto" }}>
+      <Inspector
+        request={{ method: method.name, params: exampleParams || [] }}
+        url={searchUrl && searchUrl.includes(".json") ? null : searchUrl}
```

Comment:
is this the case for rpc-discover or is this not affected by that?

Reply:
I guess should the endpoint support the discover endpoint optionally or is this just better to limit the scopefor now to be .json specific endpoints

Reply:
yea basically if it ends in `.json` i dont want to set it as a `url` for the nested `inspector`s

Reply:
It's the other way around. this sets `url: null` if searchUrl includes `.json`.